src/toast/toast.stories.tsx

```diff
@@ -88,10 +88,26 @@ export function NotificationToastsStory() {
                         >
                             Show toast with action
                         </Button>
+                        <Button
+                            variant="primary"
+                            onClick={() => {
+                                const actionLabel = getRandom(actions)
+                                showToast({
+                                    message: `${count.current++}: ${getRandom(message)}`,
+                                    action: {
+                                        label: actionLabel,
+                                        onClick: storybookAction(actionLabel),
+                                    },
+                                    showStickyToast: true,
+                                })
```

Comment:
I would suggest doing it like this:

```suggestion
                                showToast({
                                    message: `${count.current++}: ${getRandom(message)}`,
                                    action: {
                                        label: actionLabel,
                                        onClick: storybookAction(actionLabel),
                                        closeToast: false,
                                    },
                                })
```

A few things to note about this:

1. Since this is associated to the `action` no longer closing the toast, it feels more contextual to put this flag inside the action object.
2. Calling it `closeToast` is more straightforward, IMO, about what t does.
3. This new flag would, of course, be `true` by default, so that all our toasts keep the current functionality.

What do you think @nats12?

Reply:
Thanks @gnapse! Great suggestion, I didn't think to add it to the `action` object. Makes sense.

I've done that now in the last commit. Let me know if there's anything else 👍🏼